README.md

```diff
@@ -195,6 +196,73 @@ Available helper methods are:
+
+##### total_pages:
+This could be defined as the number of pages, given the limit requested. As the other values, this can be a custom number of pages, depending on your needs. For this example, this number is just an integer calculation of the total pages, depending on the limit. Also, like `@content`, we are inheriting the `@limit` variable, which allows us to operate with it however we want.
+
+##### total_count:
+This will be the number that will tell us 'how many records is returning the request'. Again, we can customize it however we want. For this particular example this will be just the count of `@content`.
+
+##### count method as delegate:
+In the end of the CustomAdapter we are delegating the `count` method to the `paginated_content`. This is because the Base Adapter delegates that method to the inherited adapter, so owr custom adapter has to know "how to calculate" that method, that's why we are defining a `count` method in the delegation (It is always mandatory to define the `count` method in a custom adapter, whether is a method definition or a delegate).
+
+If the content is an ActiveRecord_Relation, for example, this adapter would work, because `paginated_content` would become an ActiveRecord_Relation, which actually knows "how to calculate" the count method. This works as a delegate, because ActiveRecord_Relation has an internal `count` definition, but we would have to provide the needed method definition if it is a custom method, or we want a custom behaviour of a known method.
+
+##### other available methods to overwrite:
+`Wor::Paginate::Adapters::Base` also has implementations for `next_page` and `previous_page` methods (which calculate the number of the next and previous pages, respectively). If you want, you can also overwrite those methods, to calculate custom 'next' and 'previous' page numbers.
+
+To understand better the implementation of the Base Adapter and how you could overwrite methods in order to make a functional Custom Adapter, take a look at its definition in: https://github.com/Wolox/wor-paginate/blob/master/lib/wor/paginate/adapters/base.rb.
+Keep in mind that an instance of your Custom Adapter must answer `true` to the `adapt?` method inherited from the Base Adapter, in order to make it "adaptable" to the content.
+
+#### Adapters Operations
+There are also helper methods available to dynamically operate the gem's adapters, so you can configurate them, whether in the initializer or in an internal part of your application. Once you include the gem, you'll be provided with the following methods, inside the Config module:
+* `Config.add_adapter(adapter)`: Add an specific adapter to the array of the gem's adapters. The 'adapter' variable must be a Class reference to an Adapter Class (that class has to have a similar structure as the CustomAdapter example above).
```

Comment:
:thinking: `Config` with capitalize? I'm not really sure :sweat_smile: 
```suggestion
* `config.add_adapter(adapter)`: Add a specific adapter to the array of the gem's adapters. The 'adapter' variable must be a Class reference to an Adapter Class (that class has to have a similar structure as the CustomAdapter example above).
```
(same comment to lines below)

Reply:
Actually it is neither with capitalize, nor with lower case 😅 , it is the whole module, I was just putting an example. I'll change this to the proper call of the module: `Wor::Paginate::Config`